src/controller/statement.js

```diff
@@ -0,0 +1,7 @@
+const BankStatementController = { }
+
+BankStatementController.getAll = async () => {
+    // mongoose get request
+}
+
+module.exports = BankStatementController
```

Comment:
No need to fix the new line in the end of the file, but we need to configure ESLint